app/helpers/expenses_helper.rb

```diff
@@ -0,0 +1,7 @@
+module ExpensesHelper
```

Comment:
- [ ] When I try to save a category without entering a value, to would show this: 
![image](https://user-images.githubusercontent.com/58504655/227233473-25daa252-e098-4515-b140-7616cd2d1f2d.png)

Let's fix this for the app to work properly.